sample_code/ep312/rev06/t.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+# from typing import TYPE_CHECKING
+
+if False:  # TYPE_CHECKING:
```

Comment:
LoL, very bizarre haX

Reply:
pyflakes has no branch detection so this "works" :D